setup.py

```diff
@@ -9,7 +9,7 @@
     long_description = f.read()
 
 setup(
-    name="sqlalchemy-logger",
+    name="SQLAlchemyLogging",
```

Comment:
what I meant is not to change the name of the package, rather name python logger you use in sqlalchemy_logger/logger.py  "SQLAlchemyLogging". so the line 5 of sqlalchemy_logger/logger.py would be `logger = logging.getLogger(__name__)`